### PR TITLE
feat!: Remove base64 encoding

### PR DESCRIPTION
**Pull request by werne2j**
### Description
Remove the base64 encoding that is currently happening on the fly and change expectations that use provide base64 in Vault if they want to use the data field of a secret.

**This is a breaking change**

**Fixes:** #92 

### Checklist
Please make sure that your PR fulfills the following requirements:
- [x] Reviewed the guidelines for contributing to this repository
- [x] The commit message follows the [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0/#summary).
- [x] Tests for the changes have been updated
- [x] Docs have been added / updated

### Type of Change

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] New tests
- [ ] Build/CI related changes
- [ ] Documentation content changes
- [ ] Other (please describe)

### CODE REVIEW

File: pkg/kube/template.go [hunk 1]
```diff
@@ -104,50 +104,26 @@ func configReplacement(key, value string, vaultData map[string]interface{}) (int
 
 // secretReplace will replace the <placeholders> in the template's data with values from Vault.
```
Comment (jkayani) [inline pkg/kube/template.go:105]: There are some comments that refer to the old behavior that should be removed

**Review by jkayani — APPROVED** [review]
Just the minor nit to clean up some comments referring to the old behavior. Also the CHANGELOG update for v0.6.0 isn't in this in PR, would like to see it before merging

**Comment by werne2j** [comment]
Tested using https://github.com/werne2j/argocd-example/blob/master/kubernetes/example.yaml in argocd.  

Vault values:
Screen Shot 2021-03-15 at 6 41 47 PM

ArgoCD:
Screen Shot 2021-03-15 at 6 44 26 PM

Kubernetes:
Screen Shot 2021-03-15 at 6 46 26 PM

**Review by jkayani — APPROVED** [review]
Excellent, thanks!

**Pull request merged**